spec/support/feature/pikaday_support.rb

```diff
@@ -0,0 +1,11 @@
+module Feature
+  module PikadaySupport
+    def fill_in_pikaday_input(field_name, with:)
+      find_field(field_name).click
+      fill_in(field_name, with: with)
+      # This causes each Pikaday input to be correctly set
+      page.execute_script("$('body').focus()")
+      all("form").first.click
```

Comment:
👍🏽

> On 23 Jun 2016, at 21:03, Jordan Maguire <notifications@github.com> wrote:
> 
> In spec/support/feature/pikaday_support.rb:
> 
> > @@ -0,0 +1,11 @@
> > +module Feature
> > +  module PikadaySupport
> > +    def fill_in_pikaday_input(field_name, with:)
> > +      find_field(field_name).click
> > +      fill_in(field_name, with: with)
> > +      # This causes each Pikaday input to be correctly set
> > +      page.execute_script("$('body').focus()")
> > +      all("form").first.click
> I think I did...I'll try without again and annotate what happens in here
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>